### PR TITLE
Adding slack link to Contributions page

### PR DESCRIPTION
**Pull request by pvamshi**
Suggest me better phrase if this does not look good.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Please note that modifications should follow these coding guidelines:
 *   Code should pass the `eslint` linter
 *   Vertical whitespace helps readability, don’t be afraid to use it
 
+## Discussion
+Join the slack channel [linter-plugin-dev](https://atomio.slack.com/messages/linter-plugin-dev/) to discuss with fellow members.
```
Comment (Arcanemagus) [inline CONTRIBUTING.md]: How about this?

>If you have any questions during development you can join the #linter-plugin-dev channel on [Atom's Slack instance](http://atom-slack.herokuapp.com/).

A direct link to the channel won't work as users need to already have joined Atom's Slack instance or that will just lead them to a sign in page. The link above is a sign in _and_ register page.

Comment (pvamshi) [inline CONTRIBUTING.md]: Sure will change to it.

**Comment by pvamshi** [comment]
@Arcanemagus Updated the PR.

**Comment by Arcanemagus** [comment]
Works for me, and nobody else has commented so I'll merge, thanks!

**Pull request merged**